aiida/workflows/arithmetic/add_multiply.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Basic calcfunction-based workflows for demonstration purposes."""
+
+from aiida.engine import WorkChain, calcfunction, workfunction
+
+
+@calcfunction
+def add(x, y):  # pylint: disable=invalid-name
+    return x + y
+
+
+@calcfunction
+def multiply(x, y):  # pylint: disable=invalid-name
+    return x * y
+
+
+@workfunction
+def add_and_multiply(x, y, z):  # pylint: disable=invalid-name
+    """Add two numbers and multiply it with a third."""
+    addition = add(x, y)
+    product = multiply(addition, z)
+    return product
+
+
+class AddMultiplyWorkChain(WorkChain):
```

Comment:
Do we need this one as well? Where is this used? And if so, why doesn't it have an entry point like the others? Just to be clear, I prefer not to over do it with built in "useless" calculations and workchains, so if not necessary, I'd prefer to remove this one

Reply:
We don't really need it, and I'm fine with removing it. I simply added it because it was presented in the documentation in the Topics -> Workflows sections, and thought it might be nice to add all "arithmetic" examples to the code for consistency.